test/test_utils/__init__.py

```diff
@@ -42,7 +42,7 @@
 TEST_TRANSFER_S3_BUCKET = "s3://dlinfra-tests-transfer-bucket"
 
 # S3 Bucket to use to record benchmark results for further retrieving
-BENCHMARK_RESULTS_S3_BUCKET = "s3://dlinfra-dlc-cicd-performance"
+BENCHMARK_RESULTS_S3_BUCKET = "s3://bapac-chexpert-mini"
```

Comment:
Why does this bucket need to change? Please revert if this was used simply for testing purposes.

Reply:
This was just for local testing. Since at that time, I didn't have access to that bucket.

Reply:
Reverting.